vue/components/ui/molecules/select/select.vue

```diff
@@ -5,7 +5,7 @@
             class="dropdown-select"
             v-bind:value="value"
             v-if="isDevice()"
-            v-on:change="() => onSelectChange($event.target.value)"
+            v-on:change="onSelectChange($event.target.value)"

```

Comment:
Standard: don't rely on Vue callback transformation. When binding to callbacks pass a function.

```suggestion
            v-on:change="event => onSelectChange(event.target.value)"
```